docs/reference-guides/best-practices/rancher-server/tips-for-scaling-rancher.md

```diff
@@ -0,0 +1,54 @@
+
+### Managing Your Object Counts
+ETCD eventually encounters limitations to the number of a single Kubernetes resource type it can store. These exact numbers are not well documented. From internal observations we usually see performance issues once a single resource type's object count exceeds 60k, and often that type is Rolebindings.
+
+Rolebindings are created in the local cluster as a side effect of many operations.
+
+Considerations when attempting reduce rolebindings in the local cluster:
+* Only add users to clusters and projects when necessary
+* Remove clusters and projects when they are no longer needed
+* Only use custom roles if necessary
+* Use as few rules as possible in custom roles
+* Consider whether adding a role to a user is redundant
+* Consider that using less, but more powerful, clusters may be more efficient
+* Monitoring the rolebindings created from creating new projects for specific purposes versus creating new clusters
+
+### Using Apps V2 Over Apps V1
+There are two App kubernetes resources that Rancher uses: apps.projects.cattle.io and correct this apps.cattle.cattle.io. V1 here refers to apps.projects.cattle.io since they were introduced first and are legacy, while V2 refers to apps.catalog.cattle.io. Apps V2 are preferrable because they live in the downstream cluster while Apps V1 lives in the local cluster. The dashboard uses Apps V2 while the old `/g/clusters` UI uses Apps V1.
+
+### Using the Authorized Cluster Endpoint (ACE)
+There is an _Authorized Cluster Endpoint_ option for Rancher provisioned RKE1, RKE2, and K3s clusters. When enabled this adds a context to kubeconfigs generated for the cluster that uses a direct endpoint to the cluster and bypasses Rancher. However, it is not enough to only enable this option. The user of the Kubeconfig needs to use `kubectl use-context <ace context name>` in order to start using it.
+
+Without using ACE, all kubeconfig requests first route through Rancher.
+
+### Experiemntal: Option to Reduce Event Handler Executions
+The bulk of Rancher's logic occurs on event handlers. These event handlers run on an object whenever the object is updated, and when Rancher is started. Additionally, they run every 15 hours when caches are synced. In scaled setups these scheduled runs come with huge performance costs because every handler is being run on every applicable object. However, this scheduled execution of handlers can be disabled using the CATTLE_SYNC_ONLY_CHANGED_OBJECTS environment variable. If resource allocation spikes are seen on an interval of about 15 hours it is possible this setting can help.
+
+The value for the environment variable can be a comma separated list of the following options. The values refer to types of controllers (the structures that contain and run handlers) and their handlers.
```

Comment:
You should specify whether adding a type to the list enables those controllers to run, or disables them.